heidrun/ct_dc.rb

```diff
@@ -0,0 +1,95 @@
+
+# If more than one dc:contributor property exists then it drops the last one
+# If one or zero exist return nil
```

Comment:
The comment is incorrectly worded. What should be returned is essentially an empty string (non-nil value). This again gets to the heart of my question to @no-reply about how can I avoid creating empty nodes during the mapping.